docs/blog/microbit/microbitv2.md

```diff
@@ -0,0 +1,70 @@
+# MakeCode for the micro:bit V2 Release
+
+**Posted on November 20th, 2020 by [Jaqster](https://github.com/jaqster)**
+
+"Aw3s0m3!!!"
+
+"Brilliant !"
+
+"Pleeeeease, Santa?!!"
+
+"I. Want. These."
+
+"This is awesome! Our 6/yo daughter loves playing with the first micro:bit. She’s really excited about getting one with a mic and speaker."
+
+This was some of the feedback we heard from the announcement of the new micro:bit last month. And as you can see, everyone’s super excited about micro:bit V2, including us! To learn more about the new micro:bit, visit https://microbit.org/new-microbit.
+
+We’ve been working closely with the Micro:bit Educational Foundation on supporting this new hardware with MakeCode. Here’s a rundown of the new micro:bit features in MakeCode:
+
+## Compatibility
+
+We spent a lot of time making sure that whether students were using a V1 or V2 version of the micro:bit, they would have a seamless experience in MakeCode. All the MakeCode blocks will work with the new micro:bit. Some blocks (listed below), will only work with the new micro:bit, but these blocks have been clearly marked as V2 in the MakeCode Toolbox.
+
+![micro:bit V2 Toolbox](/static/blog/microbit/microbitv2/v2-toolbox.png)
+
+And when you use any of the V2 blocks in your program, you’ll see the micro:bit Simulator change to represent the new micro:bit with the "V2" indicator.
+
+![micro:bit V2 Simulator](/static/blog/microbit/microbitv2/simulator.png)
+
+Web USB reliability improvements Web USB (https://makecode.microbit.org/device/usb/webusb) isn’t a new feature, but we took this opportunity to improve the reliability of this technology. Most teachers and students have told us that the download process of copying program files over to the micro:bit is one of the most painful parts of programming with the micro:bit. Web USB is a new technology (only available in Chrome and Edge browsers) which allows you to pair your micro:bit with MakeCode allowing you to do a one-click download!
```

Comment:
```suggestion
## Web USB reliability improvements
Web USB (https://makecode.microbit.org/device/usb/webusb) isn’t a new feature, but we took this opportunity to improve the reliability of this technology. Most teachers and students have told us that the download process of copying program files over to the micro:bit is one of the most painful parts of programming with the micro:bit. Web USB is a new technology (only available in Chrome and Edge browsers) which allows you to pair your micro:bit with MakeCode allowing you to do a one-click download!
```

Reply:
you can also point to https://www.youtube.com/watch?v=_-xtP2FH9B8&list=PLMMBk9hE-SepwjCAK7cY-jvq6KeQKda8x